### PR TITLE
feat(cli): improve `migration:check` command to also print the schema diff

### PR DESCRIPTION
**Pull request by stefansundin**
I think the `migration:check` command would be better if it also reported what the diff is. I am planning to run it in CI to ensure there isn't any migration drift, and I think it is helpful to have more information when it errors.

Example output:

Screenshot 2023-05-25 at 16 46 06

This isn't super polished as I am not sure exactly how you'd prefer for it to work, so please let me know if there's any changes you'd like to see. Thanks!

### CODE REVIEW

File: packages/migrations-mongodb/src/Migrator.ts [hunk 1]
```diff
@@ -56,6 +56,10 @@ export class Migrator implements IMigrator {
     return true;
   }
 
+  async getSchemaDiff(blank: boolean, initial: boolean): Promise<{ up: string[]; down: string[] }> {
+    return { up: [], down: [] };
+  }
```
Comment (stefansundin) [inline packages/migrations-mongodb/src/Migrator.ts:61]: This is probably wrong. I don't use mongodb so I don't know what it should be.

**Comment by B4nan** [comment]
Any reason you don't just use `schema:update --dump`?

**Comment by stefansundin** [comment]
> Any reason you don't just use `schema:update --dump`?

I didn't know about that command. It looks like that should work for my use case. Feel free to close this PR then. 👍

**Comment by rubiin** [comment]
this can be closed i think

**Pull request closed without merging**